scholia/app/templates/pathway.html

```diff
@@ -86,8 +86,33 @@
   }
 }
 `
+const  listOfAuthorsSparql = `
```

Comment:
Should this maybe just be called authorsSparql so there is a one-to-one correspondence with the Javascript name in the HTML ID?

Reply:
Fixed in next PR